hhs_oauth_server/urls.py

```diff
@@ -17,7 +16,6 @@
     url(r'.well-known/', include('apps.wellknown.urls')),
     url(r'^v1/accounts/', include('apps.accounts.urls')),
     url(r'^v1/connect/userinfo', openidconnect_userinfo, name='openid_connect_userinfo'),
-    url(r'^v1/userinfo', userinfo_w_login, name='openid_connect_user_w_login'),
```

Comment:
I don't see a reason to remove an endpoint, we can mark it for deprication at some point but an API shouldn't change easily. It could possibly point to the same function, though I'm not sure there's a benifite to that in this case.

Reply:
David, there's actually a little bit of extra history here. All of our endpoints used to have two versions - one for each login type. Except that this type of login isn't actually valid for our API, as it doesn't use an access token. I previously removed the second version from all of our /fhir endpoints, and this just changes it to match.

In addition, this endpoint isn't documented anywhere. I only noticed it because I was doing a comprehensive overview of endpoints with Synack.